OperatorAssignment2.js

```diff
@@ -0,0 +1,48 @@
+// **********************************  JavaScript Task #2 Operator Problem  ***********************************************
+
+const array  = ["OR",["<","a","b"],["AND",["==","c","d"],["!=","e","f"]]]
+
+const flatedArray = array.flat(Infinity); //removing nested arrays
+let length = flatedArray.length;
+let i = length-1;
+console.log('\n');
+let bufferArray=[];
+let buff2=[];
+let j=1;
+while( i >= 0 ){ 
+    if( bufferArray.length === 3){
+        let firstElement = bufferArray[bufferArray.length -1];
+        bufferArray.pop();
+        let secondElement = bufferArray[bufferArray.length -1];
+        bufferArray.pop(); 
+        let thirdElement = bufferArray[bufferArray.length -1];
+        bufferArray.pop(); 
+        let temp ='';
+        let temp2 = temp.concat(secondElement,firstElement,thirdElement)
+        buff2.push(temp2);
+    }else if( buff2.length === 2){
+        let firstElement = buff2[buff2.length -1];
+        buff2.pop();
+        let secondElement = buff2[buff2.length -1];
+        buff2.pop(); 
+        let temp ='';
+        let temp2;
+        if(flatedArray[i] ==='AND' ){
+             temp2 = temp.concat('(',firstElement,' ',flatedArray[i],' ',secondElement,')')
```

Comment:
Same as in the last file (OperatorAssignment.js)
- const tempStr = ''.concat('(',firstElement,' ',flatedArray[i],' ',secondElement,')')
OR
- const tempStr = `(${firstElement} ${flatedArray[i] ${secondElement})}`